lib/matplotlib/collections.py

```diff
@@ -80,9 +80,7 @@ class Collection(artist.Artist, cm.ScalarMappable):
     # _offsets must be a Nx2 array!
     _offsets.shape = (0, 2)
     _transOffset = transforms.IdentityTransform()
-    _transforms = []
-
-
+    _transforms = np.empty((0, 3, 3))
```

Comment:
Since you've touched it, would you mind adding a ```#:``` style docstring for what _transforms is supposed to be - this will make reviewing (and editing) this part of the code easier in the future.